include/z64camera.h

```diff
@@ -27,6 +27,89 @@
 #define PARENT_CAM(cam) ((cam)->play->cameraPtrs[(cam)->parentCamId])
 #define CHILD_CAM(cam) ((cam)->play->cameraPtrs[(cam)->childCamId])
 
+#define CAM_HUD_ALPHA_SHIFT 8
+#define CAM_HUD_ALPHA_MASK (0x0F00)
+#define CAM_HUD_ALPHA(alpha) (((alpha) & 0xF) << CAM_HUD_ALPHA_SHIFT)
+
+#define CAM_HUD_ALPHA_50 CAM_HUD_ALPHA(0)
+#define CAM_HUD_ALPHA_1 CAM_HUD_ALPHA(1)
+#define CAM_HUD_ALPHA_2 CAM_HUD_ALPHA(2)
+#define CAM_HUD_ALPHA_3 CAM_HUD_ALPHA(3)
+#define CAM_HUD_ALPHA_4 CAM_HUD_ALPHA(4)
+#define CAM_HUD_ALPHA_5 CAM_HUD_ALPHA(5)
+#define CAM_HUD_ALPHA_6 CAM_HUD_ALPHA(6)
+#define CAM_HUD_ALPHA_7 CAM_HUD_ALPHA(7)
+#define CAM_HUD_ALPHA_8 CAM_HUD_ALPHA(8)
+#define CAM_HUD_ALPHA_9 CAM_HUD_ALPHA(9)
+#define CAM_HUD_ALPHA_A CAM_HUD_ALPHA(10)
+#define CAM_HUD_ALPHA_B CAM_HUD_ALPHA(11)
+#define CAM_HUD_ALPHA_C CAM_HUD_ALPHA(12)
+#define CAM_HUD_ALPHA_D CAM_HUD_ALPHA(13)
+#define CAM_HUD_ALPHA_IGNORE CAM_HUD_ALPHA(0xF)
+
+/**
+ * shrinkWindowFlag: determines the size of the letter-box shrink window. See CAM_SHRINKWINVAL_* enums.
+ *                  Can also add on the flag ( | CAM_SHRINKWIN_INSTANT) to make the window shrink immediately
+ * interfaceAlpha: hides certain hud icons
+ *    - A value of 0 in camera is translated to an interface alpha of 50, 
+ *      which is the value to restore all hud icons to the screen (CAM_HUD_ALPHA_50)
+ *    - A value of 0xF in camera results in no change in the alpha (CAM_HUD_ALPHA_IGNORE)
+ * funcFlags: Custom flags for functions
+ */
+#define CAM_INTERFACE_FLAGS(shrinkWindowFlag, interfaceAlpha, funcFlags) \
```

Comment:
So as seen on discord this wouldn't be "flags" because it's a bitfield where each bit isn't individually a flag/toggle thing

flags -> field, params, args, ... ? :muscle: :cat: